examples/client/index.html

```diff
@@ -88,8 +88,36 @@
 						return "Error: 'value' in 'where' clause is missing.";
 					}
 
-					// todo: find out how to decode bigint field types. i.e. what is a 34n?
-					// assert `where` type
+					const keyType = fieldTypes[whereNode.key];
+
+					const FIELD_TYPE_STRING = 1 << 0;
+					const FIELD_TYPE_NUMBER = 1 << 1;
+					const FIELD_TYPE_BOOLEAN = 1 << 4;
+					const FIELD_TYPE_NULL = 1 << 5;
+
```

Comment:
Wouldn't it be nice if we could import these constans from the Appendable module?

Or will that just be extra fluff we don't need.

Reply:
Yeah, that makes sense. We can use a typescript enum, see https://www.typescriptlang.org/docs/handbook/enums.html#computed-and-constant-members

Reply:
```ts
export enum FieldType {
  Null = 1 << 0,
  ...
}
```

will let you do `Appendable.FieldType.Null` in the js.